scss/components/_table-styles.scss

```diff
@@ -316,10 +316,24 @@ h3 + .simple-table {
 
 // Dense table
 // For tables with a lot of data that need to be condensed
+.dense-table-container {
+  box-shadow: -10px 0 20px -10px $gray-medium inset;
+  overflow-x: scroll;
+
+  @include media($lg) {
+    box-shadow: none;
+  }
+}
+
+.option__content .dense-table-container {
+  box-shadow: -10px 0 20px -10px $gray-medium inset;
+  overflow-x: scroll;
+}
+
 .dense-table {
-  table-layout: auto;
-  border-top: none;
   border-bottom-width: 0;
+  border-top: none !important;
```

Comment:
Added this above the style block with the !important rule to disable to SCSS lint and hound from flagging it:
```// scss-lint:disable ImportantRule```